inventory/src/main/java/org/flyve/inventory/categories/OperatingSystem.java

```diff
@@ -180,11 +180,11 @@ private String removeBraket(String str) {
     public static String getKernelVersion() {
         try {
             Process p = Runtime.getRuntime().exec("uname -a");
-            InputStream is = null;
+            InputStream is;
             if (p.waitFor() == 0) {
                 is = p.getInputStream();
             } else {
-                is = p.getErrorStream();
+                return "";
```

Comment:
Hi, @rafaelje 
But you need to handle the error with @thestralbot `p.getErrorStream();`